meta-xt-prod-cockpit-rcar-driver-domain/recipes-qt/examples/dual-render-3d-part.bb

```diff
@@ -0,0 +1,39 @@
+DESCRIPTION = "Qt Quick 3D demo cluster"
+
+LICENSE = "CLOSED"
+
+inherit systemd
+
+RDEPENDS_${PN} = " \
+         qtquick3d \
+         qtbase \
+         qtdeclarative \
+"  
+
+FILES_${PN} = " \
+    ${bindir}/dual-render-3d-part \
+    ${systemd_unitdir}/system/3d-part.service \
+    ${sysconfdir}/eglfs.conf \
+"
+
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
+
+SRC_URI += " \
+	file://3d-part.service \
+	git://git@gitpct.epam.com:rec-inv/dual-render-3d-part.git;branch=main \
+	file://eglfs.conf \
+"
+
```

Comment:
Please, add line
```
SRCREV = "84c7549e358ad2df3b46cb1ff8fe1dcd156c8bdf"
```
or `"${AUTOREV}"`, but in this case we need to add also `PV = "1.0+git${SRCPV}"`